openedx/core/djangoapps/profile_images/images.py

```diff
@@ -0,0 +1,130 @@
+"""
+Image file manipulation functions related to profile images.
+"""
+from cStringIO import StringIO
+
+from django.conf import settings
+from django.core.files.base import ContentFile
+from PIL import Image
+
+from ..user_api.accounts.api import get_profile_image_storage
+
+
+class DevMsg(object):
```

Comment:
These dev messages are pretty specific to file uploads. It would be nice if the class name reflected that, e.g. `FileUploadDevMessage`.